examples/seismic/model.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, origin, spacing, shape, vp, nbpml=0, dtype=np.float32,
             self.m = DenseData(name="m", shape=self.shape_domain, dtype=self.dtype)
             self.m.data[:] = self.pad(1 / (self.vp * self.vp))
         else:
-            self.m = 1/vp**2
+            self.m = ConstantData(name="m", dtype=np.float32, value=1/vp**2)
```

Comment:
We probably want a utility routine here that allocated the right type based on inspecting the argument. This way we can re-use it for all of the TTI-related fields just the same way.